.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
```

Comment:
I've not come across this kind of file before
Can I assume that it works similar to something like a `.prettierrc` file?
Is this editor independent or does it only work for vscode?

Reply:
That's correct, `root = true` is used to tell the rule processor that this `.editorconfig` file is the one and the only in this repository to stop it from traversing the whole directory structure.

Reply:
There is a VS Code plugin for it https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig. EditorConfig is supported by all major editors and IDEs.